osu.Game/Localisation/EditorStrings.cs

```diff
@@ -19,6 +19,11 @@ public static class EditorStrings
         /// </summary>
         public static LocalisableString ShowHitMarkers => new TranslatableString(getKey(@"show_hit_markers"), @"Show hit markers");
 
+        /// <summary>
+        /// "Seek to Object after placement"
+        /// </summary>
+        public static LocalisableString SeekToHitobject => new TranslatableString(getKey(@"seek_to_hitobject"), @"Seek to Object after placement");
```

Comment:
random capitalisation

```suggestion
        /// <summary>
        /// "Seek to object after placement"
        /// </summary>
        public static LocalisableString SeekToHitObject => new TranslatableString(getKey(@"seek_to_hit_object"), @"Seek to object after placement");
```

\+ update references to match